common/predictive-text/worker/tsconfig.json

```diff
@@ -4,8 +4,8 @@
     "declaration": true,
     "module": "none",
     "outFile": "../build/intermediate/index.js",
+    "inlineSourceMap": true,
```

Comment:
This allows the WebWorker to actually include the TS source.  Unfortunately, this doesn't account for the prefixed polyfills that we also bundle in, so they're probably not directly usable.

Reply:
Does this increase the final size of the .js?

Reply:
That it will; it does add a significant amount of data.

I never did figure out exactly how to serve up the map in a usable manner otherwise, though I did find a possible lead.  Keep in mind, we are looking to have a source map accessible via link from a Blob, which does make things tricky.

Reply:
> That it will; it does add a significant amount of data.

In that case I don't think we should do this.

We don't want to increase the file size for our (or even other developer's) convenience. It's already probably too big 😢. Remember that many of our users are on very bandwidth constrained devices.

If necessary, setup a debug build that inlines the sourcemap, but the release build should not .